src/app/forms/templates/general/dimensions.template.ts

```diff
@@ -0,0 +1,91 @@
+import { FormNode, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.types';
+
+export function getDimensionsSection(includeTrailer: boolean = false): FormNode {
+  const section: FormNode = {
+    name: 'dimensionsSection',
+    label: 'Dimensions',
+    type: FormNodeTypes.GROUP,
+    viewType: FormNodeViewTypes.SUBHEADING,
+    children: [
+      {
+        name: 'dimensions',
+        value: '',
+        type: FormNodeTypes.GROUP,
+        children: [
+          {
+            name: 'length',
+            label: 'Length (mm)',
+            value: '',
+            type: FormNodeTypes.CONTROL,
+            viewType: FormNodeViewTypes.STRING
+          },
+          {
+            name: 'width',
+            label: 'Width (mm)',
+            value: '',
+            type: FormNodeTypes.CONTROL,
+            viewType: FormNodeViewTypes.STRING
+          },
+          {
+            name: 'axleSpacing',
+            value: '',
+            type: FormNodeTypes.GROUP,
+            children: [
+              {
+                name: 'value',
```

Comment:
The `Axle 'x' to axle 'y' (mm)` is more complicated and the rendered value is dependant on what comes back from the database. See the table at the bottom of [this ](https://dvsa.atlassian.net/wiki/spaces/HVT/pages/18286839/View+Technical+Record)page for mappings of the values, I think we will have to use either some kind of map or enum for this.